src/Slider.tsx

```diff
@@ -48,7 +48,7 @@ export const Slider: FC<Props> = ({
   disabled,
 }) => {
   const onValueChange = useCallback(
-    ([v]: number[]) => onValueChangeProp(v),
+    ([v]: number[]) => onValueChangeProp?.(v),
```

Comment:
```suggestion
    ([v]: number[]) => onValueChangeProp(v),
```

Reply:
i wonder if this is a trivial linting rule